src/sidebar/components/Chat/MessageGroup/index.tsx

```
@@ -64,7 +69,7 @@ const MessageGroup = memo(
         preprocessMarkdown(node.content)
       ) : (
         <React.Fragment key={`message-${node.id}`}>
-          <Markdown children={parsedContent.message} />
+          <Markdown className="w-fit" children={parsedContent.message} />
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)